conf/application.conf

```diff
@@ -72,6 +72,12 @@ microservice {
           host = localhost
           port = 8203
       }
+
+      managePensions {
```

Comment:
Move this to urls section, services are internal so host will change to manage-pensions.public.mdtp which isn't accessible from outside mdtp, put this in the urls section instead